src/django/pfb_analysis/models.py

```diff
@@ -235,6 +236,7 @@ def set_boundary_file(self, geom):
             boundary_file = File(open(zip_filename, 'rb'))
             self.boundary_file = boundary_file
             self.geom = geom
+            self.geog = geom  # Also save as geography for fast lookup in the Crashes query
```

Comment:
I missed that there are actually two places where we save the boundary to the Neighborhood model--this one, where we're loading boundaries from a batch file, and in `_set_geom_from_boundary_file()`, where we load from the shapefile provided in the Neigoborhood add/edit form.  So there's should be another copy of this line in that method, down at line 331.